src/Symfony/Component/Translation/MessageCatalogue.php

```diff
@@ -220,12 +231,14 @@ public function addResource(ResourceInterface $resource)
     /**
      * {@inheritdoc}
      */
-    public function getMetadata($key = '', $domain = 'messages')
+    public function getMetadata($key = '', $domain = null)
     {
         if ('' == $domain) {
```

Comment:
So these should be a null check, and deprecate emptry string behavior (all still worth it?)